### PR TITLE
Fix/blocker works modal and sync transifex changes

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[QA][my-orcid] No error message selecting funded-by relationship type when not allowed](https://trello.com/c/WPWSOYPt/7735-qamy-orcid-no-error-message-selecting-funded-by-relationship-type-when-not-allowed)

**Comment by leomendoza123** [comment]
[[QA][my-orcid] Unable to save work with identifier that cannot be resolved](https://trello.com/c/VQCDMyfI/7732-qamy-orcid-unable-to-save-work-with-identifier-that-cannot-be-resolved)

**Comment by leomendoza123** [comment]
[[QA][my-orcid] Auto-select part-of relationship type when user selects ISSN](https://trello.com/c/JQQRpR97/7736-qamy-orcid-auto-select-part-of-relationship-type-when-user-selects-issn)

**Comment by leomendoza123** [comment]
[[QA][my-orcid] Auto-select relationship type for ISBN depending on work type](https://trello.com/c/NOTu4XV0/7737-qamy-orcid-auto-select-relationship-type-for-isbn-depending-on-work-type)

File: src/app/record/components/work-form/work-form/work-form.component.ts [hunk 1]
```diff
@@ -265,6 +274,20 @@ export class WorkFormComponent implements OnInit {
             )
           )
           formGroup.controls.externalIdentifierId.updateValueAndValidity()
+          const suggestedRelationship = this.getOrcidRecommendedRelationShip(
+            externalIdentifierType
+          )
+          console.log(externalIdentifierType)
```
Comment (DanielPalafox) [inline src/app/record/components/work-form/work-form/work-form.component.ts:280]: Could you remove the console logs

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Comment by leomendoza123** [comment]
[[QA][my-orcid] Valid DOI URL not recognized](https://trello.com/c/7yGLC6vl/7723-qamy-orcid-valid-doi-url-not-recognized)

**Pull request merged**